src/nix/info-store.cc

```diff
@@ -16,9 +16,8 @@ struct CmdPingStore : StoreCommand, MixJSON
 
     std::string doc() override
     {
-        return
-          #include "ping-store.md"
-          ;
+        return #include "info-store.md"
```

Comment:
Not a big deal (and no blocker to me), but since we're renaming things, we could rename this to `store-info.md` to match the command (`ping-store` comes from the original command name and was already out of sync)

Reply:
store-info files are renamed according to the command name

Reply:
I think what @thufschmitt is saying is that it would be good to call both files `store-info.{cc,md}`. The file was never properly renamed when the commands were made hierarchical before, but we should fix that.

Reply:
Sure.  I will try to implement the aliasing in store.cc.   Thanks